lib/author.rb

```diff
@@ -0,0 +1,28 @@
+class Author
+  attr_reader :first_name, :last_name, :books
+
+    def initialize(author_names)
+      @first_name = author_names[:first_name]
+      @last_name = author_names[:last_name]
+      @books = []
+    end
+
+    def name
+      @first_name + " " + @last_name
```

Comment:
good job doing this in a method this time, because we DO need the separate first/last names below